server/src/main/java/org/elasticsearch/repositories/Repository.java

```diff
@@ -353,4 +354,18 @@ default Set<String> getUsageFeatures() {
     static boolean assertSnapshotMetaThread() {
         return ThreadPool.assertCurrentThreadPool(ThreadPool.Names.SNAPSHOT_META);
     }
+
+    /**
+     * Get the current count of snapshots in progress
+     *
+     * @return The current number of shard snapshots in progress metric value, or null if this repository doesn't track that
+     */
+    @Nullable
+    default LongWithAttributes getShardSnapshotsInProgress() {
+        return null;
```

Comment:
I think I'd rather not have a `default` implementation here and instead just require all the non-blobstore repository implementations to return `null` explicitly. But also all the non-blobstore repository implementations are read-only, so they can reasonably return `0` here.

Reply:
Fixed in ff821584bc5bb889042fc7d5f60bc758ef70693d